composer.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "symfonycasts/sass-bundle",
+    "description": "Delightful Sass Support for Symfony + AssetMapper",
+    "license": "MIT",
+    "type": "library",
+    "keywords": ["asset-mapper", "sass"],
+    "authors": [
+        {
+            "name": "Mathéo Daninos",
+            "homepage": "https://github.com/WebMamba"
+        }
+    ],
+    "require": {
+        "php": ">=8.1",
```

Comment:
was there ultimately an extension needed for unzipping the tar?

Reply:
Looks like it's enable by default: https://www.php.net/manual/en/phar.installation.php